src/python/pants/scm/subsystems/changed.py

```diff
@@ -74,69 +75,99 @@ class ChangedOptions:
+            old_option="include_dependees",
+            new_option="dependees",
+            old_scope="changed",
+            new_scope="changed",
+            old_container=options,
+            new_container=options,
+        )
+        return cls(since, options.diffspec, dependees)
 
     def is_actionable(self) -> bool:
-        return bool(self.changes_since or self.diffspec)
+        return bool(self.since or self.diffspec)
 
     def changed_files(self, *, scm: Scm) -> List[str]:
         """Determines the files changed according to SCM/workspace and options."""
         workspace = ScmWorkspace(scm)
         if self.diffspec:
             return cast(List[str], workspace.changes_in(self.diffspec))
 
-        changes_since = self.changes_since or scm.current_rev_identifier
+        changes_since = self.since or scm.current_rev_identifier
         return cast(List[str], workspace.touched_files(changes_since))
 
 
 class Changed(Subsystem):
-    """A subsystem for global `changed` functionality.
+    """Tell Pants to detect what files and targets have changed from Git.
 
-    This supports the `--changed-*` argument target root replacements, e.g. `./pants --changed-
-    parent=HEAD~3 list`.
+    See https://pants.readme.io/docs/advanced-target-selection.
     """
 
     options_scope = "changed"
 
     @classmethod
     def register_options(cls, register):
+        register(
+            "--since",
+            type=str,
+            default=None,
+            help="Calculate changes since this Git spec (commit range/SHA/ref).",
+        )
         register(
             "--changes-since",
             "--parent",
-            "--since",
             type=str,
             default=None,
-            help="Calculate changes since this tree-ish/scm ref (defaults to current HEAD/tip).",
+            removal_version="2.1.0.dev0",
+            removal_hint=(
+                "Use `--changed-since` instead of `--changed-parent` or `--changed-changes-since`."
+            ),
+            help="Calculate changes since this tree-ish/scm ref.",
         )
         register(
             "--diffspec",
             type=str,
             default=None,
-            help="Calculate changes contained within given scm spec (commit range/sha/ref/etc).",
+            help="Calculate changes contained within a given Git spec (commit range/SHA/ref).",
+        )
+        register(
+            "--dependees",
+            type=DependeesOption,
+            default=DependeesOption.NONE,
+            help="Include direct or transitive dependees of changed targets.",
         )
         register(
             "--include-dependees",
-            type=IncludeDependeesOption,
-            default=IncludeDependeesOption.NONE,
+            type=DependeesOption,
+            default=DependeesOption.NONE,
             help="Include direct or transitive dependees of changed targets.",
+            removal_version="2.0.1.dev0",
```

Comment:
Just noticed that this is 2.0.1.dev0. Did we mean 2.1.0.dev0? cc @Eric-Arellano

Reply:
I did. I'll fix this in the followup to fix `--changed` to work with v2.